Tools/environment_install/APWikiWinInstall.ps1

```diff
@@ -0,0 +1,149 @@
+# Windows Powershell script, installs and builds Ardupilot Wiki in cygwin 
+# Most of the heavy lifting is done within cygwin shell via installdoc.sh
+# Can be run on top of existing cygwin install
+# Right click file and select "Run with Powershell"
+# Windows 10 only
+
+Import-Module BitsTransfer
+
+Write-Output "- Downloading cygwin"
+Start-BitsTransfer -Source "https://cygwin.com/setup-x86_64.exe" -Destination "$PSScriptRoot\setup-x86_64.exe"
+
+Write-Output "- Installing or updating Cygwin x64"
+Start-Process -NoNewWindow -wait -FilePath $PSScriptRoot\setup-x86_64.exe -ArgumentList "--root=C:\cygwin --no-startmenu --local-package-dir=$env:USERPROFILE\Downloads --site=http://cygwin.mirror.constant.com --packages git,make,python2,zip,mercurial --quiet-mode"
+
+#Run in cygwin:
+@'
+#!/bin/bash
+
+# Install and build Ardupilot wiki in cygwin environment
+# 
+#
+# Process can be lengthy, and default ouput from commands overwhelming 
+# (and -quiet options too silent) so we add a simple sh progress bar and spinner
+# and log to /tmp.  
+# 
+
+#set -o errexit
+#set -x
+
+function spinner {
+
+	i=1
+	sp="/-\|"
+	echo -n ' '
+	while true
+		do
+		sleep .3
+		printf "\b${sp:i++%${#sp}:1}"
+	done
+}
+
+function ProgressBar {
+# Modified from github @fearside thanks
+
+	let progress=${1}
+	let done=(${progress}*4)/10
+	let left=40-$done
+
+	fill=$(printf "%${done}s")
+	empty=$(printf "%${left}s")
+
+	printf "\r$2: [${fill// /\#}${empty// /-}] ${progress}%%  "
+
+}
+
+logfile=/tmp/wiki-install.`date +%Y-%m-%d.%H:%M:%S`.log
+
+spinner & >> $logfile 2>&1
+SPID=$! && disown
+
+printf '\nInstalling Ardupilot Wiki and build environment within Cygwin\n'
+printf '[Open 2nd Cygwin terminal and use tail -f /tmp/*.log for verbose progress output]\n\n'
+printf 'This may take several minutes ...\n\n'
+
+# Get pip
+ProgressBar 3 "Getting pip           "
+curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py > $logfile 2>&1
```

Comment:
I am a little unconfortable with those instruction as they are duplicate from the Sphinxsetup script but won't be updated if we make changes on wiki side. Can't we just use the script ?

Reply:
I agree that's a potential  issue, although the sphinxsetup script is small and simple. The problem with  using it t is that it has apt-get. Not gonna work well on cygwin ...  ;)